service/logger.go

```diff
@@ -14,6 +14,11 @@
 
 package service
 
-import logging "github.com/op/go-logging"
+import (
+	"io"
+	"log/slog"
+)
 
-var logger = logging.MustGetLogger("shadowsocks")
+func noopLogger() *slog.Logger {
```

Comment:
To make sure it doesn't create records even for Error level, you can set the Level to `slog.Level(int(slog.LevelError) +1)` or `slog.Level(math.MaxInt)`